scripts/enforce-property-single-quote-in-bundle-message

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -o pipefail -u
+
+self_dir=$(cd "$(dirname "$0")" &>/dev/null; pwd -P)
+source "$self_dir"/utils.sh
+
+
+# Find the string that have `{number}`, but have only single `'` character
```

Comment:
```suggestion
# Find the strings that have `{number}`, but contain an isolated `'` character (single single quote)
```
(`only single` suggest that we're catching the ones that have only one `'` character at all, which is not true)